tests/Feature/Framework/CustomServicesLevel/CustomModule/Level1/Level2/Level3/HappyService3.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GacelaTest\Feature\Framework\CustomServicesLevel\CustomModule\Level1\Level2\Level3;
+
+use Gacela\Framework\CustomServiceInterface;
+
+final class HappyService3 implements CustomServiceInterface
+{
+    public function greet(string $name): string
+    {
+        return "Hi, $name!";
```

Comment:
I would put maybe something like: "Hi from level #, $name", what do you think?
This way is more clear what we are doing.